app/src/main/java/com/medmapper/v33001/dto/Medicine.kt

```diff
@@ -3,13 +3,21 @@ package com.medmapper.v33001.dto
 import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
-
+/**
+ * A data class representing Medicine
+ *
+ * @property name; the name of the medicine
+ * @property strength; the strength of the medicine
+ * @property start_date; the date the patient started the medicine
+ * @property preciscription_length; how long the patient should take the medicine
```

Comment:
Added kdoc documention to better help the functionality of the data class.
https://kotlinlang.org/docs/kotlin-doc.html#block-tags